devel/apache-ant/Portfile

```diff
@@ -40,9 +42,10 @@ build.cmd               true
 variant source description "build from source; not recommended" {
         distname                        ${name}-${version}-src
         master_sites.mirror_subdir      source
-        checksums                       md5    6413ed1cee16372d0fa215ffacca48b7 \
-                                        sha1   5f0d4f059cf7dc2620cde59e9c209663e99cad54 \
-                                        sha512 5a557e06b71c69ca61bbb4d2709edb41405cb22258a158f5645cf43b1087a692a0cc9829e356238c660ed9cd917ce3ce6d8accb1775b1642545c932a49676459
+        checksums                       sha1    9e5f2400205ef25ceee2820f05b7f5bd5bbe0b00 \
```

Comment:
See other comment on sha1.

Reply:
I deliberately left it in because I left it in for the other distfile.